docs/layers/grid-layer.md

```diff
@@ -218,6 +218,86 @@ You should pass in the function defined outside the render function so it doesn'
  }
 ```
 
+
+##### `getColorWeight` (Function, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `point => 1`
+
+`getColorWeight` is the accessor function to get the weight of a point used to calcuate the color value for a cell.
+
+Note: similar to `getColorValue`, grid layer compares whether `getColorWeight` has changed to recalculate the value for each bin that its color based on.
```

Comment:
Why does this need to be called out? Does these props not behave like other accessors (i.e. recalculation is triggered by updateTriggers)?

Reply:
It will get triggered by update triggers, and behaves similar to other accessor props, nothing special. I am following the existing documentation of `getColorValue`, this can be removed.